src/css/index.css

```diff
@@ -115,16 +115,38 @@ input[type="radio"]:checked {
     z-index: 1001;
     position: absolute;
     pointer-events: none;
-    height: 100px;
+    height: auto;
     box-sizing: border-box;
     opacity: 0;
-    padding: 1.25rem;
+    padding: 0.5rem;
+    border-radius: 0.25rem;
+    border: 1px solid gray;
+
+    left: 10%;
+    top: 10%;
 
     animation-name: cell-tooltip;
     animation-duration: 0.5s;
     animation-fill-mode: forwards;
 }
 
+.cell-tooltip > table > tr > th, .cell-tooltip > table > tr > td {
+    font-size: 80%;
+    opacity: 0.8;
+}
+
+.cell-tooltip-shift-left {
+    transform: translateX(-120%);
+}
+
+.cell-tooltip-shift-top {
+    transform: translateY(-120%);
+}
+
+.cell-tooltip-shift-top-left {
+    transform: translate(-120%, -120%);
+}
```

Comment:
I don't think these are used? Instead you're tweaking the inline style? (Either approach seems fine to me, but if this is unused we should get rid of it.)

Reply:
Good catch, yes these are not used